Tasks/XamariniOS/task.json

```diff
@@ -33,11 +33,11 @@
         {
             "name": "solution",
             "aliases": [ "solutionFile" ],
-            "type": "filePath",
+            "type": "string",
```

Comment:
Why are you doing this?  This is inconsistent with other tasks and breaks compat.  The agent concats filePath types relative to the root of the repo.  String does not.  It also doesn't offer a picker.  Existing definitions that worked will break.

Reply:
+1 this should be "filePath". The task.loc.json still has "filePath", make sure you commit the generated .loc files each time.

Reply:
eric sciple on my team is a good resource to talk to.  Let's make sure this is moving in the same direction we wanted to take the msbuild task.

Reply:
Feedback from earlier in the review suggest that patterns should work when a custom "working directory" has been set for the task.  If the user had set 'src' as their working directory, the filePath type would expand a pattern like *.sln to something like D:\agent\s\*.sln when it should be looking for D:\agent\s\src\*.sln.

But it looks like you are right; previously "working directory" was ignored when looking for the solution.  I will change it back so that patterns also ignore the working directory.

Reply:
I left the original feedback for Brian after noticing that Xcode.ts passes `workingDir` as the first argument to findMatch. I now realize that filePath inputs are rooted and the first argument to findMatch is ignored for the Xcode task.